xml/MythosArea.xml

```diff
@@ -0,0 +1,10 @@
+<Panel 
+  position="124 40 -13"
+  rotation="0 0 180"
+  height="74" width="74">
+
+
+    <Button scale="0.1 0.1 1"  fontsize="62" onClick="reshuffleEncounterDeck">➡</Button>
+
+
+</Panel>
```

Comment:
Formatting is all over the place. I can recommend this VS Code extension for formatting: https://marketplace.visualstudio.com/items?itemName=DotJoshJohnson.xml